rpc/README.md

```diff
@@ -4661,6 +4663,55 @@ Response
 ```
 
 
+#### Method `get_pool_tx_detail_info`
+* `get_pool_tx_detail_info(tx_hash)`
+    * `tx_hash`: [`H256`](#type-h256)
+* result: [`PoolTxDetailInfo`](#type-pooltxdetailinfo)
+
+Query and returns the details of a transaction in the pool, only for trouble shooting
+
+###### Params
+
+*   `tx_hash` - Hash of a transaction
+
+###### Examples
+
+Request
+
+
+```
+{
+  "id": 42,
+  "jsonrpc": "2.0",
+  "method": "get_pool_tx_detail_info",
+  "params": [
+    "0xa0ef4eb5f4ceeb08a4c8524d84c5da95dce2f608e0ca2ec8091191b0f330c6e3"
+  ]
+}
+```
+
+
+Response
+
+
+```
+{
+   "jsonrpc": "2.0",
+   "result": {
+       "ancestors_count": "0x0",
+       "descendants_count": "0x0",
+       "entry_status": "Pending",
+       "pending_count": "0x1",
+       "proposed_count": "0x0",
+       "rank_in_pending": "0x1",
+       "score_sortkey": "AncestorsScoreSortKey { fee: Capacity(96942915023), weight: 274, ancestors_fee: Capacity(96942915023), ancestors_weight: 274 }",
```

Comment:
remove Capacity  : Capacity(96942915023) -> -0x5b03725df
use hex for `weight` and `ancestors_weight` is better?